Quadriga/src/main/java/edu/asu/spring/quadriga/rest/open/NodeInfoController.java

```diff
@@ -45,6 +46,7 @@
                     && occur.getProject().getProjectAccess() == EProjectAccessibility.PUBLIC) {
                 JSONObject occurance = new JSONObject();
                 occurance.append("text", occur.getTextUri());
```

Comment:
I thought of adding this also in the method but in the earlier review request this wasn't highlighted to be added in the method, so haven't added that. Will add it.